conda_lock/_vendor/conda/cli/install.py

```diff
@@ -292,11 +292,11 @@ def install(args, parser, command='install'):
                 # Other solvers might implement their own internal retry logic without
                 # depending --freeze-install implicitly like conda classic does. Example
                 # retry loop in conda-libmamba-solver:
-                # https://github.com/conda-incubator/conda-libmamba-solver/blob/da5b1ba/conda_libmamba_solver/solver.py#L254-L299
+                # https://github.com/conda/conda-libmamba-solver/blob/da5b1ba/conda_libmamba_solver/solver.py#L254-L299
```

Comment:
@jezdez I believe you use some global search and replace here, right? These two URLs won't be the same.

Reply:
Agreed, and fixed in cb682df